fenix/app/src/main/java/org/mozilla/fenix/addons/AddonPopupBaseFragment.kt

```diff
@@ -51,6 +63,35 @@ abstract class AddonPopupBaseFragment : Fragment(), EngineSession.Observer, User
                 owner = this,
                 view = view,
             )
+
+            downloadsFeature.set(
```

Comment:
Is this truly safe? DownloadsFeature holds some kind of state (in my brief look) and how does that work when there are two instances of it (`BrowserFragment` and `AddonPopupBaseFragment`)?

Reply:
We are instantiate`DownloadsFeature` in both `BrowserFragment` and `AddonPopupBaseFragment`  same happens with [promptsFeature](https://github.com/mozilla-mobile/firefox-android/pull/4845/files/3f98238951f3e470475863120997d5011516357f#diff-c1b68c76c9740c7e30d694d89f40819528bb55134ae00706bbe825b22efcb52cR39).